src/main/java/core/basesyntax/figures/Figure.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.figures;
+
+public abstract class Figure implements IPrintable, IFigure {
+    private String color;

```

Comment:
As far as we already have `Color` enum, representing colors, we can change our `color` variable type to `Color`